connector/src/main/java/org/geysermc/connector/network/UpstreamPacketHandler.java

```diff
@@ -67,8 +76,41 @@ public boolean handle(LoginPacket loginPacket) {
         return true;
     }
 
+    // Subclients login is slightly different
+    // * GeyserSession is created at this point
+    // * Some things have been done for the main client and don't need to be repeated:
+    //   * Version compatibility
+    //   * Encryption
+    //   * Resource packs
+    // Instead we can immediately connect the session
+    @Override
```

Comment:
This should also be using javadoc styled documentation. You should just be able to type in `/**` and press return and it should automatically format some of that.